tests/Zend/Validate/FloatTest.php

```diff
@@ -35,12 +35,22 @@
  */
 class Zend_Validate_FloatTest extends PHPUnit_Framework_TestCase
 {
+    /**
+     * Constant for Non-breaking space UTF-8 encoded value.
+     * https://en.wikipedia.org/wiki/Non-breaking_space
+     */
+    const NBSP = " ";
+
     /**
      * Zend_Validate_Float object
      *
      * @var Zend_Validate_Float
      */
     protected $_validator;
+    /**
```

Comment:
just a newline between docblocked vars
```suggestion

    /**
```